modules/cms-kit/src/Volo.CmsKit.MongoDB/Volo/CmsKit/MongoDB/Blogs/MongoBlogPostRepository.cs

```diff
@@ -99,4 +101,10 @@ public class MongoBlogPostRepository : MongoDbRepository<CmsKitMongoDbContext, B
         var queryable = await GetMongoQueryableAsync(token);
         return await queryable.AnyAsync(x => x.BlogId == blogId && x.Slug.ToLower() == slug, token);
     }
+
+    public async Task<List<CmsUser>> GetAuthorsHasBlogPosts(CancellationToken cancellationToken = default)
+    {
+        var queryable = (await GetQueryableAsync()).Select(x => x.Author).Distinct();
```

Comment:
I'm not sure if this query works for all conditions on MongoDB, I'll try this before merge

Reply:
Updated to: https://github.com/abpframework/abp/pull/12048/commits/df12120c4323c48f909f0ff7ee3ccfb30868b75f